Detectors/TPC/simulation/src/SAMPAProcessing.cxx

```diff
@@ -65,17 +68,18 @@ bool SAMPAProcessing::importSaturationCurve(std::string file)
   return true;
 }
 
-void SAMPAProcessing::getShapedSignal(float ADCsignal, float driftTime, std::array<float, mNShapedPoints>& signalArray)
+void SAMPAProcessing::getShapedSignal(float ADCsignal, float driftTime, std::array<float, 8> &signalArray)
```

Comment:
Array sizes need to be know at compile time. If you would like to use a member variable it would need to be `const` or even `constexpr`. Then, it cannot be changed during runtime. But this would prevent using this value from a parameter class. Otherwise one would need to use a vector.
The code in SAMPAProcessing is now a bit inconsistent due to this ... So we'll need to change the logic.